src/Pluto.jl

```diff
@@ -9,12 +9,28 @@ Have a look at the FAQ:
 https://github.com/fonsp/Pluto.jl/wiki
 """
 module Pluto
-project_relative_path(xs...) = normpath(joinpath(dirname(dirname(pathof(Pluto))), xs...))
+
+import RelocatableFolders: @path
+const ROOT_DIR = normpath(joinpath(@__DIR__, ".."))
+const FRONTEND_DIR = @path(joinpath(ROOT_DIR, "frontend"))
+const FRONTEND_DIST_DIR = let dir = joinpath(ROOT_DIR, "frontend-dist")
+    isdir(dir) ? @path(dir) : FRONTEND_DIR
```

Comment:
I'm not 100% sure on the `frontend` and `frontend-dist` stuff, hopefully this logic is actually doing the correct thing.